recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+libxml2:
+  - 2.10
```

Comment:
Do we need recipe's cbc? Can we explicitly specify the version in the host section?